01_Tutorial/01_Basics.md

```diff
@@ -80,7 +80,11 @@ try {
 
     module.exports = exports = new NewPiecePlease(Ipfs, OrbitDB)
 } catch (e) {
+  if (typeof (window) != 'undefined') {
```

Comment:
Were you getting an actual error here?

Reply:
Yeah. Node.js doesn't have window. Some sources recommend using global keyword instead of window.

Reply:
That's odd because the first try clause should not throw an error in node.js, therefore it shouldn't be trying to evaluate `window`

Can you console.log the(e) in the catch clause there to see where it's getting confused?

Reply:
Oh you mean that error. It was nothing special. I had wrong node version (12+) so it didn't finish the npm installation (Problems with leveldown module) and I was missing some modules (orbit-db for example) and it gave me an error. I downgraded to node v11.4.0 and the installation worked fine.

Reply:
But even with Node 11 - the example code was failing down to the catch clause and then barfing on `window` ?

Reply:
It worked with Node 11

Reply:
Ok, I think I'm gonna leave this as is for now, but we should probably express more clearly that Node 12 will fail because of the compilation issues in ipfs...

Reply:
So an edit to the text specifying this would resolve this PR?